### PR TITLE
[zh] Sync docs/reference/access-authn-authz/admission-controllers.md

### LINKED ISSUE #27545: [zh] Duplicate word in Using Admission Controllers
使用准入控制器
每个准入控制器的作用是什么？
PodNodeSelector
该准入控制器通过读取名字空间注解和全局配置，来为名字空间中可以可以使用的节点选择器 设置默认值并实施限制。

此处的“可以”重复了两次。

### PR DESCRIPTION
**Pull request by fenggw-fnst**
1. Update content/zh/docs/reference/access-authn-authz/admission-controllers.md
2. Fix #27545

Signed-off-by: Guangwen Feng <fenggw-fnst@cn.fujitsu.com>

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -254,6 +254,37 @@ that specifies a 'group' (or 'organization attribute') of system:masters.
...
+<!--
+This admission controller does not do anything when no default ingress class is configured. When more than one ingress
+class is marked as default, it rejects any creation of `Ingress` with an error and an administrator
+must revisit their `IngressClass` objects and mark only one as default (with the annotation
+"ingressclass.kubernetes.io/is-default-class").  This admission controller ignores any `Ingress`
+updates; it acts only on creation.
+-->
+当未配置默认 Ingress 类时，此准入控制器不执行任何操作。如果将多个 Ingress 类标记为默认 Ingress 类，
+它将拒绝任何创建 `Ingress` 的操作，并显示错误。
+要修复此错误，管理员必须重新访问其 `IngressClass` 对象，并仅将其中一个标记为默认（通过注解
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:277]: revisit 在这里是“检查”的意思

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -254,6 +254,37 @@ that specifies a 'group' (or 'organization attribute') of system:masters.
...
+"ingressclass.kubernetes.io/is-default-class"）。
+此准入控制器会忽略所有 `Ingress` 更新操作，仅响应创建操作。
+
+<!--
+See the [ingress](/docs/concepts/services-networking/ingress/) documentation for more about ingress
+classes and how to mark one as default.
+-->
+更多关于 Ingress 类，以及如何将 Ingress 类标记为默认，请参见
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:285]:
```suggestion
关于 Ingress 类以及如何将 Ingress 类标记为默认的更多信息，请参见
```

**Review by tengqm — COMMENTED** [review]
overall looks good other than some nits.

**Comment by fenggw-fnst** [comment]
Thanks for the review. Done.

**Comment by fenggw-fnst** [comment]
@tengqm Ping, PTAL, thanks.

**Pull request merged**